src/main/java/net/minestom/server/network/NetworkBufferImpl.java

```diff
@@ -10,8 +10,7 @@
 
 import javax.crypto.Cipher;
 import javax.crypto.ShortBufferException;
-import java.io.EOFException;
-import java.io.IOException;
+import java.io.*;
```

Comment:
Please prevent * imports

Reply:
Feels unnecessary (IntelliJ auto-groups imports into * if you have too many), but changed